packages/in_app_purchase/lib/src/billing_client_wrappers/sku_details_wrapper.dart

```diff
@@ -156,10 +168,48 @@ class SkuDetailsResponseWrapper {
 
     final SkuDetailsResponseWrapper typedOther = other;
     return typedOther is SkuDetailsResponseWrapper &&
-        typedOther.responseCode == responseCode &&
+        typedOther.billingResult == billingResult &&
         typedOther.skuDetailsList == skuDetailsList;
   }
 
   @override
-  int get hashCode => hashValues(responseCode, skuDetailsList);
+  int get hashCode => hashValues(billingResult, skuDetailsList);
+}
+
+/// Params containing the response code and the debug message from In-app Billing API response.
```

Comment:
nit: 

```suggestion
/// Params containing the response code and the debug message from the Play Billing API response.
```

Same suggestion applies to a few docs on this method.